### ISSUE TITLE
common-protos for ScalaPB 0.11

### ISSUE THREAD

**Issue by Tochemey** [scalapb/ScalaPB#1122]

**Comment by thesamet** [comment]
Hi @Tochemey , thanks for reporting. The release of common-protos for ScalaPB 0.11 is blocked by https://github.com/googleapis/java-common-protos/issues/134 as currently the Scala-Java converters do not compile due to a mismatch between the Java classes and protos shipped with common-protos.

**Comment by thesamet** [comment]
This is currently blocked due to https://github.com/googleapis/java-common-protos/issues/173

**Comment by Tochemey** [comment]
@thesamet it seems the blocker has been resolved https://github.com/googleapis/java-common-protos/issues/173.

**Comment by thesamet** [comment]
This is currently blocked due to https://github.com/googleapis/java-common-protos/issues/179

**Comment by Ic3fr0g** [comment]
@thesamet since this is an important release (finally able to work with Spark 3.1.2, sparksql-scalapb and common-protos) can you update the documentation in https://scalapb.github.io/docs/ ?

**Comment by Ic3fr0g** [comment]
FYI the workaround is not required now - https://github.com/googleapis/java-common-protos/pull/214

**Comment by thesamet** [comment]
@Ic3fr0g Updated the docs. Also released 2.5.0 for proto-google-common-protos which indeed doesn't require the workaround.

**Comment by Visorgood** [comment]
Hello @thesamet !
Could you please clarify, which exact library dependency should we use with Spark 3.1.2?
I am trying
```
"com.thesamet.scalapb" %% "sparksql31-scalapb0_11" % "1.0.0"
```
but I get the error:
```
found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error] 
[error] 	* org.typelevel:frameless-core_2.12:0.12.0 (early-semver) is selected over 0.10.1
[error] 	    +- org.typelevel:frameless-dataset-spark31_2.12:0.12.0 (depends on 0.12.0)
[error] 	    +- org.typelevel:frameless-dataset_2.12:0.10.1        (depends on 0.10.1)
```

**Comment by thesamet** [comment]
Hi @Visorgood , it looks like your question is unrelated to common-protos (which is the issue you have posted it on). Can you file a new issue, also can you investigate how `frameless-dataset` version 0.10.1 gets into your build? It appears to be causing the error. You are using the correct dependency for the spark version you use.

**Comment by Visorgood** [comment]
Thank you @thesamet ! You are right, the problem is a different one. I've solved it by adding exclusion for frameless-dataset library in this way:
```
excludeDependencies ++= Seq(
  ExclusionRule("org.typelevel", "frameless-dataset_2.12")
)
```
